worker/sync/db-persist/mail-contributor.js

```diff
@@ -74,8 +74,7 @@ module.exports = exports = async (event) => {
 			log.info('Some emails were not delivered. Not delivered count', res.total_rejected_recipients)
 		}
 
-		let responseData;
-		res.json().then(data => responseData = data)
+		let responseData = await res.json();
```

Comment:
improvement: Should be `const` as the variable is not reassigned
```suggestion
		const responseData = await res.json();
```